lib/workers/common.ts

```diff
@@ -104,3 +105,12 @@ export interface BranchConfig
   upgrades: BranchUpgradeConfig[];
   packageFiles?: Record<string, PackageFile[]>;
 }
+
+export interface AssigneesAndReviewersConfig {
```

Comment:
I'm not sure based on which criteria these config interfaces are split. Happy to improve this.